### PR TITLE
Tidy up UeberBackend a bit

### PR DESCRIPTION
**Pull request by zeha**
### Short description
Code cleanup only. May improve compilation times.

### Checklist
I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [X] compiled
- [ ] and tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added regression tests
- [ ] added unit tests

### CODE REVIEW

File: modules/remotebackend/unixconnector.cc [hunk 1]
```diff
@@ -22,6 +22,9 @@
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/un.h>
```
Comment (Habbie) [inline modules/remotebackend/unixconnector.cc]: commit message does not mention this - was this added intentionally?

File: pdns/ueberbackend.cc [hunk 2]
```diff
@@ -29,6 +29,7 @@
 #include "utility.hh"
 
 
+#include <dlfcn.h>
```
Comment (Habbie) [inline pdns/ueberbackend.cc]: was this added intentionally?

Comment (zeha) [inline pdns/ueberbackend.cc]: this was just moved from ueberbackend.hh

File: modules/remotebackend/unixconnector.cc (hunk 1 above)
Comment (zeha) [inline modules/remotebackend/unixconnector.cc]: they used to be in ueberbackend.hh, but nothing else appears to need them.

Comment (Habbie) [inline modules/remotebackend/unixconnector.cc]: Ah yes, works for me.

File: pdns/ueberbackend.cc (hunk 2 above)
Comment (Habbie) [inline pdns/ueberbackend.cc]: :+1:

**Pull request merged**